### PR TITLE
added training of model with logistic regression and result analysis …

### PR DESCRIPTION
**Pull request by thosgt**
…functions

maybe I will add tests later. Can you try to clone the repo and run it with a short dataset ?

### CODE REVIEW

File: src/results_analysis.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+def get_exercise_code_betas(coefs: pd.DataFrame):
+    exercise_codes = [
+        f"exercise_code_{exercise_code}"
+        for exercise_code in get_available_exercise_codes(coefs)
+    ]
+    return coefs[coefs["columns"].isin(exercise_codes)].sort_values("coefs")
+
+
+def get_one_exercise_gamma(
```
Comment (ghost) [inline src/results_analysis.py]: i would name it `get_gamma_of_exercise`

File: src/results_analysis.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                    "columns"
+                ].values,
+            )
+        )
+    )
+
+
+def get_exercise_gammas_of_one_exercise_code(coefs: pd.DataFrame, exercise_code: str):
+    available_exercise_codes = get_available_exercise_codes(coefs)
+    assert exercise_code in available_exercise_codes
```
Comment (ghost) [inline src/results_analysis.py]: it's weird to have an assert in the middle of a function like this. maybe if this assertion is not verified, you still want the program to run?
maybe 
```
if exercise_code not in available_exercise_codes:
    (print something?)
    return
```

File: src/results_analysis.py (hunk 1 above)
Comment (thosgt) [reply src/results_analysis.py]: done

File: src/results_analysis.py (hunk 2 above)
Comment (thosgt) [reply src/results_analysis.py]: Done

**Review by ghost — APPROVED** [review]

**Pull request merged**